test/e2e/alertmanager_test.go

```diff
@@ -2140,7 +2140,6 @@ func testAMWeb(t *testing.T) {
 func testAlertManagerMinReadySeconds(t *testing.T) {
 	// Don't run Alertmanager tests in parallel. See
 	// https://github.com/prometheus/alertmanager/issues/1835 for details.
-	runFeatureGatedTests(t)
```

Comment:
should we delete the `runFeatureGatedTests()` func and references to `FEATURE_GATED_TESTS` in the GH action workflow as well?

Reply:
No strong opinion as long as we document that it's not used for now.